src/boot/lib/ustring.ml

```diff
@@ -279,6 +279,10 @@ module Op = struct
 
   let uc c = int_of_char (if c = '\x0D' then '\x0A' else c)
 
+  let is_lower_alpha c = uc 'a' <= c && c <= uc 'z'
+
+  let is_upper_alpha c = uc 'A' <= c && c <= uc 'Z'
+
```

Comment:
As I understand it, a `uchar` represents a Unicode character. Therefore I think these functions should be named something like `is_ascii_lower_alpha` and `is_ascii_upper_alpha` to clarify that these functions only return true for ascii-characters (there are more Unicode characters that could be classified as lowercase/uppercase).

Reply:
Makes sense, I renamed them like you suggested.